IFC4x3/PropertyEnumerations/j/PEnum_JointRelativePosition.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_JointRelativePosition" Name="PEnum_JointRelativePosition" UniqueId="a83c1187-4f56-4bc6-ad43-39a30df74794">
```

Comment:
This looks familiar. Could you not generalize an enum that looks like this for example and reuse:
- `LEFT`
- `RIGHT`
- `MIDDLE`
- `UP`
- `DOWN`
and name it PEnum_RelativePosition?